pilot/pkg/bootstrap/options.go

```diff
@@ -128,13 +129,17 @@ type ValidationOptions struct {
 }
 
 var podNamespaceVar = env.RegisterStringVar("POD_NAMESPACE", "", "")
+var serviceAccountVar = env.RegisterStringVar("SERVICE_ACCOUNT", "", "")
```

Comment:
I don't think the service account name has been configurable in the past. 
The main purpose of this PR is to allow pre-1.5 sidecars to work with 1.5 istiod. We never shipped
a version if istio that used a custom service account or spiffee name.

I would just remove this, and use a constant with the expected name.

Reply:
It is configurable, but maybe not something we need to support? https://github.com/istio/istio/blob/5a16e559769e1fe3d41b30378b0649eb428db1b5/pilot/cmd/pilot-agent/main.go#L728